packages/opentelemetry-core/src/utils/trace-flags.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TraceFlags } from '@opentelemetry/api';
+
+/**
+ * Given a bitmask, determine if the SAMPLED flag is set
+ *
+ * @param traceFlags 8 bit mask of tracing flags
+ */
+export function isSampled(traceFlags: TraceFlags = TraceFlags.UNSAMPLED) {
```

Comment:
Just curious to know, why do you have default value here?

Reply:
Also, maybe we don't need a separate function for this. WDYT?

Reply:
I added this as a separate function because in a few places the logic was actually incorrect in a non-obvious way. Centralizing the logic ensures this doesn't happen in the future.

The previous logic was basically `if (traceFlags === TraceFlags.SAMPLED)` which is actually incorrect because if, for example, the traceflags integer was `3`, the `sampled` bit is still set (`00000011`), but `3 != 1`. Using the logic implemented here, you get `(3 & 1) === 1` which is correct (`00000011 & 00000001 = 00000001`)

Reply:
> I added this as a separate function because in a few places the logic was actually incorrect in a non-obvious way. Centralizing the logic ensures this doesn't happen in the future.

Ok

Reply:
Since this is now only used in a single place I decided to remove it.